examples/tutorials/colabs/Habitat_Lab.ipynb

```diff
@@ -125,7 +125,7 @@
    "metadata": {},
```

Comment:
Can we use another config here? We cannot have a tutorial use a test asset.

Reply:
Used benchmark config closest to referenced test config + config overrides